server/api/member.go

```diff
@@ -91,6 +96,19 @@ func getMembers(svr *server.Server) (*pdpb.GetMembersResponse, error) {
 			continue
 		}
 		m.GitHash = gitHash
+		found := false
+		for dcLocation, serverIDs := range dclocationDistribution {
+			for _, serverID := range serverIDs {
+				if serverID == m.MemberId {
+					m.DcLocation = dcLocation
+					found = true
+					break
+				}
+			}
+			if found {
```

Comment:
What is the `found` used for? how about directly `return members, nil` if `serverID == m.MemberId`?

Reply:
I think we can't. We should fulfill `dcLocation` for every member.

Reply:
oh, my mistake~